server/routes/api/teas.js

```diff
@@ -40,7 +41,10 @@ router.post("/new-tea", (req, res) => {
   });
 });
 
-router.put("/update-tea", (req, res) => {
+// @route PUT api/teas
+// @desc Update tea
+// @access Public
+router.put("/", (req, res) => {
```

Comment:
Did you decide to keep the ID out of this `PUT` path, or was this an oversight? REST spec states you should be getting ID from `params`, not `body`.